leapp/cli/upgrade/__init__.py

```diff
@@ -14,6 +17,40 @@
 from leapp.utils.output import report_errors, beautify_actor_exception
 
 
+def archive_logfiles():
+    """ Archive log files from a previous run of Leapp """
+    archive_dir = '/var/log/leapp/archive/'
+    debug_dir = '/var/log/leapp/dnf-debugdata/'
+    leapp_db = '/var/lib/leapp/leapp.db'
+    log_files = [
+        '/var/log/leapp/leapp-upgrade.log',
+        '/var/log/leapp/leapp-report.txt',
+        '/var/log/leapp/dnf-plugin-data.txt'
+    ]
+    files_to_archive = [file for file in log_files if os.path.isfile(file)]
+
+    if os.path.isdir(debug_dir) and os.listdir(debug_dir):
+        files_to_archive.append(debug_dir)
+
+    if not os.path.isdir(archive_dir):
+        os.makedirs(archive_dir)
+
+    if files_to_archive:
+        archive_file = os.path.join(archive_dir, 'leapp-{}-logs.tar.gz'.format(int(time.time())))
```

Comment:
change the time.time() to something else. Now it returns number of second since 1.1.1970.
E.g. `leapp-1555445048-logs.tar.gz`

Reply:
Take a look at https://github.com/oamg/leapp/pull/494